osconfig_tests/utils/utils.go

```diff
@@ -20,6 +20,10 @@ import (
 	"math/rand"
 	"time"
 
+	"github.com/GoogleCloudPlatform/compute-image-tools/osconfig_tests/compute"
+
```

Comment:
remove this line

Reply:
i think i need to fix my intellij. i used to work on vim and vim was super great at formatting and managing imports. i guess i need to setup intellij to accommodate it